src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -115,15 +115,11 @@ public function dump($input, $inline = 0, $indent = 0, $flags = 0)
                 if ($value instanceof TaggedValue) {
                     $output .= sprintf('%s%s !%s', $prefix, $dumpAsMap ? Inline::dump($key, $flags).':' : '-', $value->getTag());
 
-                    if ($inline - 1 <= 0) {
+                    if ($inline - 1 <= 0 || is_scalar($value->getValue())) {
```

Comment:
should `null` also be dumped in the short way ? If yes, it needs to be handled here (`is_scalar` returns `false` for `null`)

Reply:
Yes, makes sense.